$NetBSD: patch-src_Makefile.am,v 1.7 2020/05/23 17:42:32 fox Exp $

Removed tcl8.6, asound and jack flags, this is now handled in the pkgsrc
script.

--- src/Makefile.am.orig	2020-05-19 18:46:10.000000000 +0000
+++ src/Makefile.am
@@ -1,5 +1,5 @@
-AM_CXXFLAGS = -I ../include -I /usr/include/tcl8.6 -Wall -D_THREAD_SAFE -DHAVE_OPENGL -D__LICENSED__ -D __SVG__ -D__GPL20__ -D __BOOST_TIME__ -DPREFIX=\"@prefix@\"
-LIBS +=  -ltcl8.6 -lSDL -lGL -lpthread -lasound
+AM_CXXFLAGS = -I ../include -Wall -D_THREAD_SAFE -DHAVE_OPENGL -D__LICENSED__ -D __SVG__ -D__GPL20__ -D __BOOST_TIME__ -DPREFIX=\"@prefix@\"
+LIBS +=  -lSDL -lGL -lpthread -lasound
 
 AM_CFLAGS = -I ../include
 bin_PROGRAMS = din
